pkgs/top-level/all-packages.nix

```diff
@@ -22235,6 +22235,9 @@ in
 
   pybitmessage = callPackage ../applications/networking/instant-messengers/pybitmessage { };
 
+  wrapPyspread = libsForQt5.callPackage ../applications/misc/pyspread-app/wrap-pyspread.nix { };
```

Comment:
Expose only a single attribute, the application that works.

Reply:
When writing a NixOS module or a home-manager, should I call the file `wrap-pyspread.nix` directly?

Reply:
I modified the program to make `pyspread-app` a set, and move the executable application to `pyspread-app.app`.